scripts/aem.js

```diff
@@ -612,6 +612,21 @@ async function loadBlocks(main) {
   }
 }
 
+/**
```

Comment:
Please see if we can avoid changes in aem.js (which is to be considered as equivalent to a readonly library) and refactor them elsewhere.

Reply:
done, removed it completely by using the existing readblockconfig method